n8n.json

```
@@ -0,0 +1,44 @@
+{
+  "n8n": {
+    "containers": {
+      "n8n": {
+        "image": "n8nio/n8n",
+        "launch_order": 1,
+        "ports": {
+          "5678": {
+            "description": "n8n web UI",
+            "label": "Web UI Port",
+            "protocol": "tcp",
+            "host_default": 9099,
+            "ui": true
+          }
+        },
+        "volumes": {
+          "/home/node/.n8n": {
+            "description": "Persistent n8n data (workflows, credentials, settings)",
+            "label": "n8n Data [e.g. n8n-data]"
+          }
+        },
+        "opts": [
+          ["-e", "N8N_BLOCK_ENV_ACCESS_IN_NODE=true"],
+          ["-e", "N8N_ENFORCE_SETTINGS_FILE_PERMISSIONS=true"],
+          ["-e", "DB_SQLITE_POOL_SIZE=4"],
+          ["-e", "N8N_RUNNERS_ENABLED=true"],
+          ["-e", "N8N_DIAGNOSTICS_ENABLED=false"],
+          ["-e", "N8N_VERSION_NOTIFICATIONS_ENABLED=false"],
+          ["-e", "N8N_GIT_NODE_DISABLE_BARE_REPOS=true"],
+          ["-e", "N8N_SECURE_COOKIE=false"]
+         ]
+      }
+    },
+    "description": "n8n is a workflow automation tool. It allows you to connect apps and services, automate tasks, and create complex workflows with a visual editor. Available for amd64 amd arm architecture",
```

```suggestion
    "description": "n8n is a workflow automation tool. It allows you to connect apps and services, automate tasks, and create complex workflows with a visual editor.<p>Based on the <a href='https://hub.docker.com/r/n8nio/n8n' target='_blank'>official docker image</a>, available for amd64 amd arm architecture",
```

Adding the docker image link in here.